src/domain/ticketSystemPassword/mutations/TicketSystemPasswordMutations.ts

```diff
@@ -0,0 +1,20 @@
+import { gql } from '@apollo/client';
+
+export const importEventTicketSystemPasswordsMutation = gql`
+  mutation ImportTicketSystemPasswordsMutation(
+    $input: ImportTicketSystemPasswordsMutationInput!
+  ) {
+    importTicketSystemPasswords(input: $input) {
+      event {
+        id
+        name
+      }
+      passwords
```

Comment:
Are these actually needed?

Reply:
This also relates to the "TODO: generate" -comment in the `ticketSystemPasswordsApi.ts`. This model should update the generated schema.